build on master failing - "type annotations required"

Build on `master` appears to be failing right now.
This is after doing `rustup update`.

```
error[E0283]: type annotations required: cannot resolve `_: keychain::Keychain`
   --> src/bin/grin.rs:449:18
    |
449 |                 let wallet = FileWallet::new(wallet_config.clone(), "").unwrap_or_else(|e| {
    |                              ^^^^^^^^^^^^^^^
    |
    = note: required by `<wallet::FileWallet<K>>::new`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0283`.
error: Could not compile `grin`.
```

--- a/src/bin/grin.rs
+++ b/src/bin/grin.rs
@@ -371,7 +371,12 @@ fn server_command(server_args: Option<&ArgMatches>, mut global_config: GlobalCon
 		info!(
 			LOGGER,
 			"Starting the Grin server from configuration file at {}",
-			global_config.config_file_path.as_ref().unwrap().to_str().unwrap()
+			global_config
+				.config_file_path
+				.as_ref()
+				.unwrap()
+				.to_str()
+				.unwrap()
 		);
 		global::set_mining_mode(
 			global_config
@@ -446,9 +451,10 @@ fn server_command(server_args: Option<&ArgMatches>, mut global_config: GlobalCon
 		let _ = thread::Builder::new()
 			.name("wallet_owner_listener".to_string())
 			.spawn(move || {
-				let wallet = FileWallet::new(wallet_config.clone(), "").unwrap_or_else(|e| {
-					panic!("Error creating wallet: {:?} Config: {:?}", e, wallet_config)
-				});
+				let wallet: FileWallet<ExtKeychain> = FileWallet::new(wallet_config.clone(), "")
+					.unwrap_or_else(|e| {
+						panic!("Error creating wallet: {:?} Config: {:?}", e, wallet_config)
+					});
 				wallet::controller::owner_listener(wallet, "127.0.0.1:13420").unwrap_or_else(|e| {
 					panic!(
 						"Error creating wallet api listener: {:?} Config: {:?}",

